app/views/categories/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render partial: 'shared/navbar', locals: {heading: 'Categories', move: ''} %>
+<%= link_to 'LogOut', destroy_user_session_path,  data: {
+    turbo_method: :delete,
+    turbo_confirm: "Are you sure?"
+    }, class: "special-btn" %>
+<div class="main-category-container hide-on-med-and-up">
+    <div class="category-container center-align">
+        <%= render partial: 'list', collection: @categories, as: :category %>
+    </div>
+    <a id="add-cat-btn" class="waves-effect waves-light btn-small hoverable" href="/categories/new"><i class="material-icons right">create</i>Add Category</a>
```

Comment:
- [ ] [OPTIONAL] Well done for making changes to the colors as suggested by the previous reviewer. Please also consider making the buttons look exactly like the one in the given design.
### Your Design
![image](https://github.com/Shakir-Hussain12/Snapscan/assets/46839548/07a18537-b157-42ff-80fa-1c2d04031474)
### Design Guideline
![image](https://github.com/Shakir-Hussain12/Snapscan/assets/46839548/8acfe4a6-3327-406e-9b6a-fd12d1f522b4)

Reply:
will do, thanks